com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsRendering.cs

```diff
@@ -129,7 +130,25 @@ internal static DebugUI.Widget CreateMapOverlaySize(DebugDisplaySettingsRenderin
                 getter = () => (int)data.wireframeMode,
                 setter = (value) => { },
                 getIndex = () => (int)data.wireframeMode,
-                setIndex = (value) => data.wireframeMode = (WireframeMode)value
+                setIndex = (value) => data.wireframeMode = (WireframeMode)value,
+                onValueChanged = (_, _) => DebugManager.instance.ReDrawOnScreenDebug()
+            };
+
+            internal static DebugUI.Widget CreateWireframeNotSupportedWarning(DebugDisplaySettingsRendering data) => new DebugUI.Container
+            {
+                nameAndTooltip = Strings.WireframeNotSupportedWarning,
+                isHiddenCallback = () =>
+                {
+                    switch (SystemInfo.graphicsDeviceType)
```

Comment:
That's a good idea. I believe Editor itself only runs on platforms that support wireframe rendering, so build target platform is more relevant when user sees this warning in the editor.

Reply:
I ended up doing something a bit different here. Instead of looking at build targets, I only show this warning in player builds. AFAIK this is never needed in the editor, only Android Vulkan/GLES have this problem. I think it's better that way you only really care about this information when you are using the debug feature on the device where it might not work correctly.